apps/src/templates/manageStudents/ManageStudentsGenderCell.jsx

```diff
@@ -40,11 +40,11 @@ class ManageStudentGenderCell extends Component {
         )}
         {this.props.isEditing && (
           <select
+            style={{width: 120, marginBottom: 0}}
```

Comment:
This doesn't matter immediately, as we are still using the Legacy version of this file, but I wanted to update here just in case that changes.